src/azul/requests.py

```diff
@@ -20,3 +21,9 @@ def requests_session_with_retry_after() -> requests.Session:
     session.mount('http://', adapter)
     session.mount('https://', adapter)
     return session
+
+
+def parse_url_qs(url):
```

Comment:
If this has universal value it should get a docstring and maybe some doctests. But I doubt it does have universal value.

Reply:
It did not seem to have universal value, i place it back within the `test_pagination.py` file.